packages/tools/src/utilities/cine/playClip.ts

```diff
@@ -330,7 +333,16 @@ function _createStackViewportCinePlayContext(
       // It is always in acquired orientation
       return true;
     },
+    tries: 0,
```

Comment:
can we extract the CINE changes into a separate PR and dedicate this PR only to stackPrefetch?